src/client/components/app/Transactions/DepositTx.tsx

```diff
@@ -54,8 +53,10 @@ export const DepositTx: FC<DepositTxProps> = ({
   const [amount, setAmount] = useState('');
   const [debouncedAmount, isDebouncePending] = useDebounce(amount, 500);
   const [txCompleted, setTxCompleted] = useState(false);
+  const [isFetchingAllowance, setIsFetchingAllowance] = useState(false);
   const currentNetwork = useAppSelector(NetworkSelectors.selectCurrentNetwork);
   const walletNetwork = useAppSelector(WalletSelectors.selectWalletNetwork);
+  const walletIsConnected = useAppSelector(WalletSelectors.selectWalletIsConnected);
```

Comment:
nit: Usually the best practice is to just have these like below
```suggestion
  const isWalletConnected = useAppSelector(WalletSelectors.selectWalletIsConnected);
```
No action needed, but would be quicker to see that that returns a boolean

Reply:
Yeah, agree with this, just used the same naming from the selector, which probably should also be named as you describe

Reply:
changed here https://github.com/yearn/yearn-finance-v3/pull/630/commits/6f98c168328fc60e05d693fbaadc3c2a0a309621